static/src/.eslintrc.js

```diff
@@ -34,6 +34,10 @@ module.exports = {
     'generator-star-spacing': 0,
 
     // allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0
+    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
+
+    'template-curly-spacing': 'off',
+
+    indent: 'off'
```

Comment:
why is `indent` turned off?

Reply:
To be honest I don't know, but yarn lint doesn't work if we remove that

Reply:
There is an issue that we should see how to fixed it seems to be related to the babel-lint version

Reply:
For now maybe we can have that, I think

Reply:
should we add a follow up for this then?